packages/lesswrong/components/common/MetaInfo.jsx

```diff
@@ -6,7 +6,7 @@ import classNames from 'classnames'
 
 const styles = (theme) => ({
   root: {
-    display: "inline",
+    display: "inline-block",
```

Comment:
Revert so we can merge the rest of this, and think about this change later and separately:
```suggestion
    display: "inline",
```